File: AF+SD+自带下拉刷新案例/YFTabBar/YFTabBar/Controller/OneViewController.m
//
//  OneViewController.m
//  YFTabbarController
//
//  Created by kangbing on 16/5/31.
//  Copyright © 2016年 kangbing. All rights reserved.
//

#import "OneViewController.h"
#import "TwoViewController.h"

@interface OneViewController ()

@end

@implementation OneViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    //
    
        
    self.view.backgroundColor = [UIColor greenColor];
        
    UIButton *btn = [UIButton buttonWithType:UIButtonTypeCustom];
        
    [btn setImage:[UIImage imageNamed:@"plusX_Last"] forState:UIControlStateNormal];
        
    btn.bounds = CGRectMake(0, 0, 64, 64);
        
    btn.center = self.view.center;
        
    [self.view addSubview:btn];
        
    [btn addTarget:self action:@selector(close) forControlEvents:UIControlEventTouchUpInside];
    
}




-(void)close{
    
    TwoViewController *tvc = [[TwoViewController alloc]init];
    
    [self.navigationController pushViewController:tvc animated:YES];
    
}


@end

File: 自定义TabBar 适配X  Swift/YFTabBar/YFTabBarController/Classes/Controllers/OneViewController.swift
//
//  OneViewController.swift
//  YFTabBarController
//
//  Created by yangfan on 2018/3/15.
//  Copyright © 2018年 yangfan. All rights reserved.
//

import UIKit

class OneViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        
        view.backgroundColor = UIColor.blue
        
        addOneBtn()
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}

extension OneViewController {
    
    func addOneBtn(){
        
        let btn = UIButton(type: UIButtonType.custom)
        
        btn.setImage(UIImage(named:"plusX_Last"), for: UIControlState.normal)
        
        btn.bounds = CGRect(x: 0, y: 0, width: 64, height: 64)
        
        btn.center = view.center
        
        btn.addTarget(self, action:#selector(close) , for: UIControlEvents.touchUpInside)
        
        view.addSubview(btn)
    }
    
    @objc func close(){
        
        navigationController?.pushViewController(TwoViewController(), animated: true);
        
    }
    
    
    
}


